dev/README.md

```diff
@@ -1,70 +1,61 @@
 # Development Tomcat Environment
 
+It is possible to use Docker containers to test your code, but it is normally more convenient to simply run the repository webapp (**`alfresco.war`**) and Share webapp (**`share.war`**) in a tomcat instance. Options are also available to apply selected AMPs.
```

Comment:
A few changes such as this one include very minimal rewording (e.g.: in this case just adding a final `.` to the sentence) at the cost of losing the formatting that is more convenient to easily scan and maintain the documentation.

Please let's retain the original formatting / try to not go overboard with the length of a single line. Apply a vertical guide line via your IDE to do so if needed, to not break the previous one.

This comment applies to ALL changes in this PR.